src/utils/form.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
These are not utils, but components you will reuse many times. They should be moved to components with proper file and folder names, like components/formFields